README.md
Iain Johnston e-portfolio
=====

This is my e-portfolio. The home page is meant to act like a terminal would and the available commands are links to further information on me.


changelog.txt
Change Log

-3/4/2014 -m 'Initial Commit'
	-- Added  all text files
	-- Added all php files
	-- Added index.html
	-- Added js and min.js files 
	-- Added css files for console.

-4/4/2014 -m 'Fixes and updates'
	-- Added References.html (links not finished page)
	-- Resized footer on nano pages
	-- Aligned sitemap links
	-- Added text to Current_Studies.php, Skills.php, Strenghts.php and Hobbies.php.
	-- Added image to Intro.php
	-- Added css file for nano pages

-6/4/2014 -m Added text and sitemap changes
	-- Formatted sitemap
	-- Finished Strengths
	-- Added SDC image to Qualifications
	-- Added sitemap to footer
	-- Added help on index.html for users who don't want to use
	   terminal.
	-- Added references.html and references to it.
	-- Added LinkedIn link to Intro.php
	-- Added to Why Cardiff
	
-7/4/2014 -m Added clear and some formatting
	-- Added functionality to clear index.html
	-- Spelling fixes
	-- Some formatting on nano pages


Skills.php
<!DOCTYPE html>
<html>
	<head>
		<link href="nanoPages.css" type="text/css" rel="stylesheet">

<meta charset=utf-8 />

<title>Skills</title>

</head>
	<body>
		<div class='header'>
			<h3 id='title' >GNU nano v2.3.4</h3>
			<h3 id='file' >File: Skills.txt</h3>
		</div>

		<div class = 'container'>
			
			<?php
				echo "<div class='fileTextSmaller'>";
				echo "<pre>";

				$handle = fopen("Files/Skills.txt", "r");
				if ($handle) {
 					while (($line = fgets($handle)) !== false) {
     					echo $line;
 					}
				} else {
 					echo "Error opening file!";
				}
				fclose($handle);
				echo "</pre>";
				echo "</div>";
 			?>
 			
 			<a href = "http://github.com/iainrj/">(1)Logic game &nbsp;&nbsp;&nbsp; </a>
 			<a href = "http://github.com/iainrj/BasesSchedule">(2)Baseball Scheduler&nbsp;&nbsp;&nbsp; </a>
 			<a href = "http://github.com/iainrj/Pass">(3)Password Generator&nbsp;&nbsp;&nbsp;</a>
 			<a href = "http://github.com/iainrj/MidnightMusic">(4)Music Store&nbsp;&nbsp;</a>
 				
		</div>

		<footer>
			<table BORDER=0 CELLSPACING=0 WIDTH="100%" >
			<tr >
			<!-- INTRO BLOCK -->
			<td align="left" WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td align="left" WIDTH="8%" ><tt><font color="#440077">&nbsp;<a href="Intro.php">Intro</a></font></tt></td>

			<!-- CURRENT STUDIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Current_Studies.php">Current Studies</a></font></tt></td>
			
			<!-- SKILLS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#02AF02" ></td>

			<td WIDTH="8%" ><tt><font color="#440077">&nbsp;<a href="Skills.php">Skills</a></font></tt></td>
			
			<!-- QUALIFICATIONS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Qualifications.php">Qualifications</a></font></tt></td>

			<!-- STRENGTHS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="10%" ><tt><font color="#440077">&nbsp;<a href="Strengths.php">Strengths</a></font></tt></td>

			<!-- HOBBIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ><b><tt><font color="#0D0D0D"></font></tt></b></td>

			<td WIDTH="13%"><tt><font color="#440077">&nbsp;<a href="Hobbies.php">Hobbies</a></font></tt></td>

			<!-- SITEMAP BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="2035D6" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="sitemap.html">Sitemap</a></font></tt></td>

			<!-- TERMINAL BLOCK -->
			
			<td align="right" WIDTH="1.5%" BGCOLOR="RED" ><b><tt><font color="#0D0D0D"></font></tt></b></td>

			<td align="right" WIDTH="8%" id="backToTerminal"><tt><font color="#440077">&nbsp;<a href="../index.html">Back to Terminal</a></font></tt></td>
			</tr>

			</table>
		</footer>

	</body>
</html>

Hobbies.php
<!DOCTYPE html>
<html>
	<head>
		<link href="nanoPages.css" type="text/css" rel="stylesheet">

<meta charset=utf-8 />

<title>Hobbies</title>

	</head>
	
	<body>
		<div class='header'>
			<h3 id='title'>GNU nano v2.3.4</h3>
			<h3 id='file'>File: Hobbies.txt</h3>
		</div>

		<div class = 'container'>
			<?php
				echo "<div class='fileTextSmaller'>";
				echo "<pre>"; 
				$handle = fopen("Files/Hobbies.txt", "r");
				if ($handle) {
 					while (($line = fgets($handle)) !== false) {
     					echo $line;
 					}
				} else {
 					echo "Error opening file!";
				}
				fclose($handle);
				echo "</pre>";
				echo "</div>";
 			?>

		</div>
			
		<footer>
			<table BORDER=0 CELLSPACING=0 WIDTH="100%" >
			<tr >
			<!-- INTRO BLOCK -->
			<td align="left" WIDTH="1.5%" BGCOLOR="FFFFFF" ></td>

			<td align="left" WIDTH="8%"><tt><font color="#440077">&nbsp;<a href="Intro.php">Intro</a></font></tt></td>

			<!-- CURRENT STUDIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Current_Studies.php">Current Studies</a></font></tt></td>

			<!-- SKILLS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="8%" ><tt><font color="#440077">&nbsp;<a href="Skills.php">Skills</a></font></tt></td>

			<!-- QUALIFICATIONS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Qualifications.php">Qualifications</a></font></tt></td>

			<!-- STRENGTHS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="10%" ><tt><font color="#440077">&nbsp;<a href="Strengths.php">Strengths</a></font></tt></td>

			<!-- HOBBIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#02AF02" ></td>

			<td WIDTH="13%"><tt><font color="#440077">&nbsp;<a href="Hobbies.php">Hobbies</a></font></tt></td>

			<!-- SITEMAP BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="2035D6" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="sitemap.html">Sitemap</a></font></tt></td>
			
			<!-- TERMINAL BLOCK -->

			<td align="right" WIDTH="1.5%" BGCOLOR="RED" ><b><tt></tt></b></td>

			<td align="right" WIDTH="8%" id="backToTerminal"><tt><font color="#440077">&nbsp;<a href="../index.html">Back to Terminal</a></font></tt></td>
			</tr>

			</table>
		</footer>

	</body>
</html>

Strengths.php
<!DOCTYPE html>
<html>
	<head>
		<link href="nanoPages.css" type="text/css" rel="stylesheet">

<meta charset=utf-8 />

<title>Strengths</title>
</head>
	<body>
		<div class='header'>
			<h3 id='title'>GNU nano v2.3.4</h3>
			<h3 id='file'>File: Strengths.txt</h3>
		</div>

		<div class = 'container'>
			
			<?php
				echo "<div class='fileText'>";
				echo "<pre>"; 
				$handle = fopen("Files/Strengths.txt", "r");
				if ($handle) {
 					while (($line = fgets($handle)) !== false) {
     					echo $line;
 					}
				} else {
 					echo "Error opening file!";
				}
				fclose($handle);
				echo "</pre>";
				echo "</div>";
 			?>
 			
		</div>

		<footer>
			<table BORDER=0 CELLSPACING=0 WIDTH="100%" >
			<tr >
			<!-- INTRO BLOCK -->
			<td align="left" WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td align="left" WIDTH="8%"><tt><font color="#440077">&nbsp;<a href="Intro.php">Intro</a></font></tt></td>

			<!-- CURRENT STUDIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Current_Studies.php">Current Studies</a></font></tt></td>

			<!-- SKILLS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="8%" ><tt><font color="#440077">&nbsp;<a href="Skills.php">Skills</a></font></tt></td>

			<!-- QUALIFICATIONS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Qualifications.php">Qualifications</a></font></tt></td>

			<!-- STRENGTHS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#02AF02" ></td>

			<td WIDTH="10%" ><tt><font color="#440077">&nbsp;<a href="Strengths.php">Strengths</a></font></tt></td>

			<!-- HOBBIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="13%"><tt><font color="#440077">&nbsp;<a href="Hobbies.php">Hobbies</a></font></tt></td>

			<!-- SITEMAP BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="2035D6" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="sitemap.html">Sitemap</a></font></tt></td>
			
			<!-- TERMINAL BLOCK -->

			<!-- <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>  Remove to add more tabs -->

			<td align="right" WIDTH="1.5%" BGCOLOR="RED" ><b><tt></tt></b></td>

			<td align="right" WIDTH="8%" id="backToTerminal"><tt><font color="#440077">&nbsp;<a href="../index.html">Back to Terminal</a></font></tt></td>
			</tr>

			</table>
		</footer>	

	</body>
</html>

Qualifications.php
<!DOCTYPE html>
<html>
	<head>
		<link href="nanoPages.css" type="text/css" rel="stylesheet">

<meta charset=utf-8 />

	<title>Qualifications</title>
	</head>
	
	<body>
		<div class='header'>
			<h3 id='title' >GNU nano v2.3.4</h3>
			<h3 id='file' >File: Qualifications.txt</h3>	
		</div>

		<div class = 'container'>
			
			<?php
				echo "<div class='fileText'>";
				echo "<pre>";
				echo "<img src='Images/sdc_crest.jpg' align='right' alt='St Dunstans College Crest' id='sdc_crest'>";

				$handle = fopen("Files/Qualifications.txt", "r");
				if ($handle) {
 					while (($line = fgets($handle)) !== false) {
     					echo $line;
 					}
				} else {
 					echo "Error opening file!";
				}
				fclose($handle);
				echo "</pre>";
				echo "</div>";
 			?>

		</div>

		<footer>
			<table BORDER=0 CELLSPACING=0 WIDTH="100%" >
			<tr >
			<!-- INTRO BLOCK -->
			<td align="left" WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td align="left" WIDTH="8%"><tt><font color="#440077">&nbsp;<a href="Intro.php">Intro</a></font></tt></td>

			<!-- CURRENT STUDIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Current_Studies.php">Current Studies</a></font></tt></td>

			<!-- SKILLS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="8%" ><tt><font color="#440077">&nbsp;<a href="Skills.php">Skills</a></font></tt></td>

			<!-- QUALIFICATIONS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#02AF02" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Qualifications.php">Qualifications</a></font></tt></td>

			<!-- STRENGTHS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="10%" ><tt><font color="#440077">&nbsp;<a href="Strengths.php">Strengths</a></font></tt></td>

			<!-- HOBBIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="13%"><tt><font color="#440077">&nbsp;<a href="Hobbies.php">Hobbies</a></font></tt></td>

			<!-- SITEMAP BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="2035D6" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="sitemap.html">Sitemap</a></font></tt></td>
			
			<!-- TERMINAL BLOCK -->

			<td align="right" WIDTH="1.5%" BGCOLOR="RED" ><b><tt></tt></b></td>

			<td align="right" WIDTH="8%" id="backToTerminal" ><tt><font color="#440077">&nbsp;<a href="../index.html">Back to Terminal</a></font></tt></td>
			</tr>

			</table>
		</footer>

	</body>
</html>

Current_Studies.php
<!DOCTYPE html>
<html>
	<head>
		<link href="nanoPages.css" type="text/css" rel="stylesheet">

<meta charset=utf-8 />

<title>Current Studies</title>
	</head>

	<body>
		<div class='header'>
			<h3 id='title'>GNU nano v2.3.4</h3>
			<h3 id='file'>File: CurrentStudies.txt</h3>
		</div>

		<div class = 'container'>
				<?php
					echo "<div class='fileText'>"; 
					echo "<pre>";
					echo "<img src='Images/CardiffLogo.png' align='right' id='CardiffLogo'>";
					
					$handle = fopen("Files/Current_Studies.txt", "r");
					if ($handle) {
     					while (($line = fgets($handle)) !== false) {
         					echo $line;
     					}
     					echo "<br/>";
     					echo "<center><a href='http://www.cs.cf.ac.uk/currentstudents/bsccompsci'>More detailed information on the modules can be found here</a></center>";
 					} else {
     					echo "Error opening file!";
 					}
 					fclose($handle);
 					echo "</pre>";
 					echo "</div>";
 				?>

		</div>

		<footer>
			<table BORDER=0 CELLSPACING=0 WIDTH="100%" >
			<tr >	
			<!-- INTRO BLOCK -->
			<td align="left" WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td align="left" WIDTH="8%"><tt><font color="#440077">&nbsp;<a href="Intro.php">Intro</a></font></tt></td>
			
			<!-- CURRENT STUDIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#02AF02" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Current_Studies.php">Current Studies</a></font></tt></td>

			<!-- SKILLS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="8%"><tt><font color="#440077">&nbsp;<a href="Skills.php">Skills</a></font></tt></td>

			<!-- QUALIFICATIONS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%"><tt><font color="#440077">&nbsp;<a href="Qualifications.php">Qualifications</a></font></tt></td>

			<!-- STRENGTHS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="Strengths.php">Strengths</a></font></tt></td>

			<!-- HOBBIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="13%"><tt><font color="#440077">&nbsp;<a href="Hobbies.php">Hobbies</a></font></tt></td>

			<!-- SITEMAP BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="2035D6" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="sitemap.html">Sitemap</a></font></tt></td>
			
			<!-- TERMINAL BLOCK -->

			<td align="right" WIDTH="1.5%" BGCOLOR="RED" ><b></td>

			<td align="right" WIDTH="8%" id="backToTerminal"><tt><font color="#440077">&nbsp;<a href="../index.html">Back to Terminal</a></font></tt></td>
			</tr>

			</table>
		</footer>

	</body>
</html>

Intro.php
<!DOCTYPE html>
<html>
	<head>
		<link href="nanoPages.css" type="text/css" rel="stylesheet">

<meta charset=utf-8 />

<title>Intro</title>

	</head>
	
	<body>
		<div class='header'>
			<h3 id='title'>GNU nano v2.3.4</h3>
			<h3 id='file'>File: Intro.txt</h3>
		</div>
		
		<div class = 'container'>
				<div class='Intro'>
					<br/>
					<img src="Images/IainJohnston" alt="Iain Johnston image" align="center">
					<br/>
					<br/>
					<table align='center' border='10em'>
						<tr></tr>
						<td align="left">Name: Iain Johnston</td>
						<tr></tr>
						<td align="left">Age: 19</td>
						<tr></tr>
						<td align="left">Email: iainrj18@gmail.com</td>
						<tr></tr>
						<td align="left">LinkedIn:  <a href="http://linkedin.com/profile/view?id=334891761">LinkedIn</a></td>
						<tr></tr>
						<td align="left">Github:  <a href='http://github.com/iainrj'>My Github</a></td>
						<tr></tr>
					</table>
					
					<br/>
					<br/>

				<center>
				<p>
					This e-portfolio should demonstrate and describe my skills. It will also give a basic overview 
					of my qualifications and hobbies.
					
				</p>
				</center>

			</div>
		</div>

		<footer>
			<table BORDER=0 CELLSPACING=0 WIDTH="100%" >
			<tr >
			<!-- INTRO BLOCK -->
			<td align="left" WIDTH="1.5%" BGCOLOR="#02AF02" ></td>

			<td align="left" WIDTH="8%"><tt><font color="#">&nbsp;<a href="Intro.php">Intro</a></font></tt></td>

			<!-- CURRENT STUDIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Current_Studies.php">Current Studies</a></font></tt></td>

			<!-- SKILLS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="8%" ><tt><font color="#440077">&nbsp;<a href="Skills.php">Skills</a></font></tt></td>

			<!-- QUALIFICATIONS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="15%" ><tt><font color="#440077">&nbsp;<a href="Qualifications.php">Qualifications</a></font></tt></td>

			<!-- STRENGTHS BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH="10%" ><tt><font color="#440077">&nbsp;<a href="Strengths.php">Strengths</a></font></tt></td>

			<!-- HOBBIES BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="#FFFFFF" ></td>

			<td WIDTH= "13%"><tt><font color="#440077">&nbsp;<a href="Hobbies.php">Hobbies</a></font></tt></td>

			<!-- SITEMAP BLOCK -->

			<td WIDTH="1.5%" BGCOLOR="2035D6" ></td>

			<td WIDTH="10%"><tt><font color="#440077">&nbsp;<a href="sitemap.html">Sitemap</a></font></tt></td>
			
			<!-- TERMINAL BLOCK -->

			<td align="right" WIDTH="1.5%" BGCOLOR="RED" ><b><tt></tt></b></td>

			<td align="right" WIDTH="8%" id="backToTerminal" ><tt><font color="#440077">&nbsp;<a href="../index.html">Back to Terminal</a></font></tt></td>
			</tr>

			</table>
		</footer>
			
	</body>
</html>